packages/apps/esm-primary-navigation-app/src/components/change-language/change-language-link.extension.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Button, SwitcherItem } from '@carbon/react';
+import { capitalize } from 'lodash-es';
```

Comment:
```suggestion
import capitalize from 'lodash-es/capitalize';
```

A minor detail I hadn't noticed in my initial review. Do you mind changing it in `ChangeLanguageModal` as well? This import syntax gives us slightly better tree-shakeability.

Reply:
sure. on it.

Reply:
Should I also move `capitalize` import  just before `@openmrs/esm-framework` import in `changeLanguageModal`?